src/main/java/org/zone/commands/structure/region/flags/entry/monster/ZoneFlagMonsterEntryEnabledCommand.java

```diff
@@ -0,0 +1,70 @@
+package org.zone.commands.structure.region.flags.entry.monster;
+
+import net.kyori.adventure.text.Component;
+import org.jetbrains.annotations.NotNull;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.configurate.ConfigurateException;
+import org.zone.commands.system.ArgumentCommand;
+import org.zone.commands.system.CommandArgument;
+import org.zone.commands.system.arguments.operation.ExactArgument;
+import org.zone.commands.system.arguments.simple.BooleanArgument;
+import org.zone.commands.system.arguments.zone.ZoneArgument;
+import org.zone.commands.system.context.CommandContext;
+import org.zone.region.Zone;
+import org.zone.region.flag.FlagTypes;
+import org.zone.region.flag.entity.monster.move.PreventMonsterFlag;
+import org.zone.utils.Messages;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class ZoneFlagMonsterEntryEnabledCommand implements ArgumentCommand {
+
+    public static final ZoneArgument ZONE_VALUE = new ZoneArgument("zoneId",
+                                                                   new ZoneArgument.ZoneArgumentPropertiesBuilder());
+    public static final BooleanArgument ENABLED = new BooleanArgument("enabledValue", "enable",
+                                                                      "disable");
+
+    @Override
+    public @NotNull List<CommandArgument<?>> getArguments() {
+        return Arrays.asList(new ExactArgument("region"),
+                             new ExactArgument("flag"),
+                             ZONE_VALUE,
+                             new ExactArgument("monster"),
```

Comment:
Mind moving the Monster argument under Entry?

Reply:
did it long ago